### PR TITLE
fix(oomstore): fix unit test

### PR DESCRIPTION
**Pull request by lianxmfor**

### CODE REVIEW

File: pkg/oomstore/online_query_test.go [hunk 1]
```diff
@@ -219,6 +219,7 @@ func prepareFeatures(isConsistent bool, isAvailable bool) types.FeatureList {
 	features := types.FeatureList{
 		{
 			Name:      "model",
+			FullName:  "user.model",
```
Comment (jinghancc) [inline pkg/oomstore/online_query_test.go:222]: "device.model" would be better

File: pkg/oomstore/online_query_test.go [hunk 2]
```diff
@@ -231,6 +232,7 @@ func prepareFeatures(isConsistent bool, isAvailable bool) types.FeatureList {
 		},
 		{
 			Name:      "price",
+			FullName:  "user.price",
```
Comment (jinghancc) [inline pkg/oomstore/online_query_test.go:235]: "device.price"

File: pkg/oomstore/feature.go [hunk 3]
```diff
@@ -104,10 +104,11 @@ func (s *OomStore) CreateFeature(ctx context.Context, opt types.CreateFeatureOpt
 }
 
 func validateFeatureFullNames(names []string) error {
+	fmt.Println("feature names:", names)
 	for _, name := range names {
 		nameSlice := strings.Split(name, FeatureFullNameSeparator)
 		if len(nameSlice) != 2 {
-			return errors.Errorf("invalid feature full name %s", name)
+			return errors.Errorf("invalid feature full name =%s=", name)
```
Comment (jinghancc) [inline pkg/oomstore/feature.go:111]: `=%s=` is this a typo?

File: pkg/oomstore/online_query_test.go (hunk 1 above)
Comment (lianxmfor) [reply pkg/oomstore/online_query_test.go]: fixed

File: pkg/oomstore/online_query_test.go (hunk 2 above)
Comment (lianxmfor) [reply pkg/oomstore/online_query_test.go]: fixed

File: pkg/oomstore/feature.go (hunk 3 above)
Comment (lianxmfor) [reply pkg/oomstore/feature.go]: fixed

**Review by YikSanChan — APPROVED** [review]
lgtm

**Pull request merged**